docs/src/components/ComponentReadme.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-danger */
```

Comment:
Why add this? I don't see `dangerouslySetInnerHTML` anywhere?
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-danger.md

Reply:
Oh yeah removed that later.